specs.json

```
@@ -878,7 +878,13 @@
   },
   "https://www.w3.org/TR/compute-pressure/",
   "https://www.w3.org/TR/contact-picker/",
-  "https://www.w3.org/TR/core-aam-1.2/",
+  {
+    "url": "https://www.w3.org/TR/core-aam-1.2/",
+    "nightly": {
+      "repository": "https://github.com/w3c/aria",
+      "sourcePath": "accname/index.html"
```

```suggestion
      "sourcePath": "core-aam/index.html"
```
shouldn't it be "core-aam/index.html" instead?

---

Ahem. Let's say that was a PR review test ;)

---

Yeah, I passed!